arcgis-ios-sdk-samples/Geometry/Cut geometry/README.md

```diff
@@ -0,0 +1,18 @@
+# Cut geometry
+
+This sample demonstrates how to cut a geometry with a polyline using the `AGSGeometryEngine`. For example, you may want to subdivide a large parcel polygon into several smaller parcels.
+
+![](CutGeometry.png)
+
+## How to use the sample
+Press the `Cut` button, and the solid blue polygon will be cut into two polygons - a yellow and a green polygon with a diagonal pattern.
+
+## How it works
+The `AGSGeometryEngine.cutGeometry:withCutter:` method is executed with the polygon as the first parameter and the cutting line as the second parameter. This returns a list of geometries. Each geometry is added as a new `AGSGraphic` with differing fill symbols to visualize the results.
```

Comment:
Please change `AGSGeometryEngine.cutGeometry:withCutter:` to `AGSGeometryEngine.cut(_:withCutter:)`. The same on line `15`.

FYI: You can use Copy Symbol Name (⌃⇧⌘C) or Copy Qualified Symbol Name (⌃⌥⇧⌘C) to easily get the name of a method.